JASP-Engine/JASP/R/commonsummarystatsttestbayesian.R

```diff
@@ -1604,6 +1613,10 @@
         xlim[1] <-  min(-2, .qShiftedT(0.01, parameters, oneSided="left"))
         xlim[2] <- max(2,.qShiftedT(0.99, parameters, oneSided="left"))
       }
+      
+      if(any(abs(xlim) == Inf)){ # In case qShiftedT fails
```

Comment:
Well, no, but this check applies only if oneSided != FALSE, so either I would do:

```
if(oneSided == FALSE){
  stuff
} else{
  if(oneSided == "right"){
    stuff
  } else if(oneSided == "left"){
    stuff
  }

  if(any(is.infinite(xlim))) stop("Cannot plot the posterior - possibly too concentrated near 0.")
}
```

or move this check below the computation of xlim and with the following:

```
if(oneSided != FALSE && any(is.infinite(xlim)))
  stop("Cannot plot the posterior - possibly too concentrated near 0.")
```

Both options seem to me less readable, but I am open to suggestions :)

Reply:
I think line 1583-1620 can be simplified to:

```
    if (oneSided == FALSE) {
      xlim[1] <- min(-2, quantile(delta, probs = 0.01)[[1]])
      xlim[2] <- max(2, quantile(delta, probs = 0.99)[[1]])
    } else {
      if (oneSided == "right")
        oneSidedDelta <- delta[delta >= 0]
      else # left
        oneSidedDelta <- delta[delta <= 0]
        
      xlim[1] <- min(-2, quantile(oneSidedDelta, probs = 0.01)[[1]])
      xlim[2] <- max(2, quantile(oneSidedDelta, probs = 0.99)[[1]])

      if (any(is.na(xlim))) {
        xlim[1] <- min(-2, .qShiftedT(0.01, parameters, oneSided=oneSided))
        xlim[2] <- max(2, .qShiftedT(0.99, parameters, oneSided=oneSided))
      }
      
      if (any(is.infinite(xlim))) # In case qShiftedT fails
        stop("Cannot plot the posterior - possibly too concentrated near 0.")
    }
```

Reply:
yeah, that's probably the best way, will do!